tests/SimplifyTest.elm

```diff
@@ -1496,8 +1496,8 @@ a = not >> not
                     |> Review.Test.run ruleWithDefaults
                     |> Review.Test.expectErrors
                         [ Review.Test.error
-                            { message = "Unnecessary double negation"
-                            , details = [ "Composing `not` with `not` cancel each other out." ]
+                            { message = "Unnecessary double Basics.not"
+                            , details = [ "Composing Basics.not with Basics.not cancels each other out." ]
```

Comment:
I don't know what the correct spelling should be here: "cancel" or "cancels"? 🤔

Reply:
Yeah I also don't like either option. The change was mostly because I had to change the error infos anyway. I'll add this to the "to rework list" :+1: